### DISCUSSION TITLE
New Model for whisper

### DISCUSSION THREAD

**Discussion by Dnkhatri** [SubtitleEdit/subtitleedit#6489] [General]
https://github.com/openai/whisper/discussions/397

reading this thread new model for whisper seems to improves the error rate.

**Comment by niksedk** [comment]
Re-downloading the "large" model should get the new "large" model via whisper.cpp in SE.
If you're using SE with whisper OpenAI then get latest beta of SE.

Yes, the new large model works really well in the one test I did :)

**Comment by ard-av** [comment]
Greetings. Delivered version SE 3.6.10 Beta. Loaded the large Whisper model. I understand correctly that the GPU is not used in this case, but the CPU is used?
![Screenshot_1](https://user-images.githubusercontent.com/114281218/206704008-a999965e-dc26-4277-bff8-9752c35e45e6.png)
![Screenshot_2](https://user-images.githubusercontent.com/114281218/206704013-1e5e9650-c152-4272-8732-d8cae450d47d.png)
![Screenshot_3](https://user-images.githubusercontent.com/114281218/206704014-15b3a40d-ca79-445c-940b-52fc2642f0ba.png)

**Reply by niksedk** [reply]
Yes, Whisper.cpp uses only cpu (AVX2): see https://github.com/ggerganov/whisper.cpp

**Reply by ard-av** [reply]
Thanks for the answer
